src/main/webapp/WEB-INF/views/EditUserPage.html

```diff
@@ -0,0 +1,176 @@
+            <button type="button" class="navbar-toggle" data-toggle="collapse"
+                    data-target="#responsive-menu">
+                <span class="sr-only">Menu</span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+            </button>
+            <a class="navbar-brand" href="#">Great<i class="fa fa-rocket fa-lg" aria-hidden="true">
+                <text/>
+            </i>Start</a>
+        </div>
+        <div class="collapse navbar-collapse" id="responsive-menu">
+            <ul class="nav navbar-nav navbar-left">
+                <li class="active"><a href="#">Home</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Events</a></li>
+                <li><a href="#">Projects</a></li>
+                <li>
+                    <form role="search" class="navbar-form navbar-left">
+                        <div class="form-group">
+                            <input type="text" placeholder="Search" class="form-control">
+                        </div>
+                    </form>
+                </li>
+            </ul>
+            <ul class="nav navbar-nav navbar-right">
+                <li class="dropdown active "><a class="dropdown-toggle" data-toggle="dropdown">
+                    <span class="pull-left" style="margin-right:8px; margin-top:-5px;">
+                        <img src="http://vev.ru/uploads/images/00/07/64/2011/10/12/c26b1b.jpg"
+                             class="img-responsive img-circle" title="UserName"
+                             alt="UserName" width="30px" height="30px"/>
+                    </span>
+                    <span>UserName</span>
+                    <b class="caret"></b></a>
+                    <ul class="dropdown-menu">
+                        <li>
+                            <div class="navbar-content">
+                                <div class="row">
+                                    <div class="col-md-5 col-xs-5">
+                                        <img src="http://vev.ru/uploads/images/00/07/64/2011/10/12/c26b1b.jpg"
+                                             alt="AltText" class="img-responsive"
+                                             width="120px" height="120px"/>
+                                    </div>
+                                    <div class="col-md-7 col-xs-7" style="margin-left: -35px">
+                                        <div class="container">
+                                            <div>Welcome back,<strong> UserName!</strong></div>
+                                            <div class="container"
+                                                 style="margin-top:5px; font-size: 15px; font-weight: 500">
+                                                <ul class="non-marker">
+                                                    <li><a href="#"><i class="fa fa-user-o" aria-hidden="true"
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)